Workshop 5/Task 2/Server.java

```diff
@@ -0,0 +1,52 @@
+import java.io.*;
```

Comment:
why there are two servers there?
We only one server and need unlimited clients, so try to do something with only one server and one client!

Reply:
### HINT :  try threading!